client/js/jsonedit.js

```diff
@@ -278,6 +277,19 @@ const jeCommands = [
       jeSetAndSelect(null);
     }
   },
+  {
+    id: 'je_duplicateWidget',
+    name: '✨ duplicate widget',
+    forceKey: 'D',
+    show: _=>jeStateNow,
+    call: function() {
+      var currentWidget = JSON.parse(JSON.stringify(jeWidget.state))
```

Comment:
Doesn't make a difference here but I feel like it's better to use `let` instead of `var` unless you actually want/need the weird behavior of `var`.